File: Unify.Android/app/src/main/java/com/owlcreativestudio/unify/models/FacebookLogin.java
package com.owlcreativestudio.unify.models;

public class FacebookLogin {
    private String facebookId;

    public FacebookLogin(String facebookId) {
        this.facebookId = facebookId;
    }

    public String getFacebookId() {
        return facebookId;
    }

    public void setFacebookId(String facebookId) {
        this.facebookId = facebookId;
    }
}

File: Unify.Server/src/Unify.Api/ViewModels/FacebookLogin.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using Unify.Common.Interfaces;

namespace Unify.Api.ViewModels
{
    public class FacebookLogin: IFacebookLogin
    {
        public string FacebookId { get; set; }
    }
}
